lib/scenario.js

```diff
@@ -81,9 +81,8 @@ module.exports.injected = function (fn, suite, hookName) {
     }
     recorder.catch((err) => {
       event.emit(event.test.failed, suite, err); // emit
-      if (hookName === 'after' || hookName === 'afterSuite') {
-        event.emit(event.hook[hookName].failed, suite);
-      }
+      if (hookName === 'after') event.emit(event.test.after, test);
```

Comment:
Should be if (hookName === 'after') event.emit(event.test.after, suite);. Test is undefined